msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -484,59 +479,50 @@ class NativeAuthPublicClientApplication(
                             channel = result.challengeChannel
                         )
                     }
+                    is SignInCommandResult.PasswordRequired -> {
+                        Logger.warn(
```

Comment:
The log message should include the context from the method signature. We can use `Logger.warn(TAG, "Sigin using password received unexpected result $result")